R/zzz.R

```diff
@@ -1,5 +1,27 @@
 .lockbox_env <- new.env()
 
+.get_safely_from_envir <- function(target, default = NULL) {
+  res <- tryCatch(
+    get(target, envir = .lockbox_env)
+  , error = function(e) {default})
+}
+
+set_session_id <- function() {
+  if (is.null(.get_safely_from_envir("session_id"))) {
+    if (requireNamespace("uuid", quietly = TRUE)) {
+      assign("session_id", uuid::UUIDgenerate(), envir = .lockbox_env)
+    } else {
+      assign("session_id", digest::digest(as.integer(Sys.time())), envir = .lockbox_env)
+    }
+  }
+  
+  # We don't want to clutter up ~/.R/ with tons of symlinked dirs, lets try to do some housekeeping
```

Comment:
I would prefix with another `#` which will get placed on the left hand side of the [rocco documentation](http://robertzk.github.io/lockbox/).